alphabet.py

```diff
@@ -0,0 +1,29 @@
+from graph import Graph
+
+
+class Alphabet:
+    def __init__(self):
+        self.letters = set()
+        self.relations = Graph()
+
+    def _compare_words(self, next_word, prev_word):
+        if len(prev_word) == 0:
+            return
+
+        for i in range(min(len(next_word), len(prev_word))):
+            p = prev_word[i]
+            n = next_word[i]
+
+            if p != n:
+                if n not in self.letters:
+                    self.letters.add(n)
+                self.relations.add(p, n)
+
+    def get_alphabet(self, unknown_dictionary):
+        prev_word = ''
+
+        for word in unknown_dictionary:
+            self._compare_words(word, prev_word)
+            prev_word = word
+
+        return self.relations.start_topological_sort()
```

Comment:
(This comment is based on the ones I wrote for start_topological_sort() ) Since you are not passing a root element, is it necessary to give it as a possible input to start_topological_sort ?
It looks to me that start_topological_order is doing some unnecessary steps, like calling _get_random_node, which seems that it will always have to be called, since no root is ever passed.
Can you tell me of a case in which a root element would be passed to start_topological_sort() ?